unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/NewProject/NewProjectSecondStepView.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+namespace DCL.Builder
+{
+    public class NewProjectSecondStepView : BaseComponentView
+    {
+        private const int MAX_PARCELS = 32;
+        private const int PARCEL_METERS_SIZE = 16;
```

Comment:
`Configuration` already contains this data, cannot we use it here?

Reply:
Yes! The max parcel is specific for builder so I will let it here